website/docs/r/nsxt_edgegateway_dhcpv6.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_edgegateway_dhcpv6"
+sidebar_current: "docs-vcd-resource-nsxt-edge-dhcpv6"
+description: |-
+  Provides a resource to manage DHCPv6 configuration for NSX-T Edge Gateways.
+---
+
+# vcd\_nsxt\_edgegateway\_dhcpv6
+
+Provides a resource to manage DHCPv6 configuration for NSX-T Edge Gateways.
+
+## Example Usage (DHCPv6 mode)
+
+```hcl
+resource "vcd_nsxt_edgegateway_dhcpv6" "dhcpv6-mode" {
+  org             = "cloud"
+  edge_gateway_id = data.vcd_nsxt_edgegateway.testing-in-vdc.id
+
+  enabled = true
+  mode    = "DHCPv6"
+}
+```
+
+## Example Usage (SLAAC mode)
+
+```hcl
+resource "vcd_nsxt_edgegateway_dhcpv6" "slaac-mode" {
+  org             = "datacloud"
+  edge_gateway_id = data.vcd_nsxt_edgegateway.testing-in-vdc.id
+
+  enabled      = true
+  mode         = "SLAAC"
+  domain_names = ["non-existing.org.tld", "fake.org.tld"]
+  dns_servers  = ["2001:4860:4860::8888", "2001:4860:4860::8844"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Required) Org in which the NSX-T Edge Gateway is located
+* `edge_gateway_id` - (Required) NSX-T Edge Gateway ID
+* `enabled` - (Required) Boolean flag if DHCPv6 is enabled or disabled. **Note**
```

Comment:
Unfinished "Note"?

Reply:
Possibly I put this by error. I don't think there is any important note to be set here